src/components/RepoBox/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import {StarIcon, RepoForkedIcon, IssueOpenedIcon} from '@primer/octicons-react';
```

Comment:
Never seen this icons plugin, looks really useful

Reply:
I think it's made by GitHub themselves, it was pretty good for getting the same icons as on the GitHub repos page